MewsSystems/mews-flutter
# feat(optimus): [RND-142516] Updated button colors for the dark theme

RND-142516

#### Summary

- Updated warning button style for the active state.
- Updated badge style to match the DS.


<details><summary>Screenshots</summary>

![CleanShot 2023-01-30 at 23 47 34](https://user-images.githubusercontent.com/9210422/215613935-a82706ce-7452-4d9a-9706-be1dbb30da65.gif)



</details>

#### Testing steps

- Open Button story
- Warning button text color should change depending on the active state

#### Follow-up issues

None

#### Check during review

- Verify against YouTrack issue.
- Is the PR over 300 additions? Consider rejecting it with advice to split it. Is it over 500 additions? It should definitely be rejected.
- Unused code removed.
- Build passing.
- Is it a bug fix? Check that it is covered by a proper test (unit or integration).
